src/components/SkipLink/SkipLink.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Meta, StoryObj } from "@storybook/react";
```

Comment:
it feels a little strange that the doc for this component is almost just a blank page. Should we add some text here to describe the relationship between this component and `AppMain`? Maybe something like the below:

```
This component is intended to be used alongside the AppMain component, with its href attribute pointing to the ID of AppMain.
```

Also, is it possible to add a `main` element to the doc with `id` linked up to `href` of `SkipLink`?

Reply:
Actually, there is no particular relationship between this component and `AppMain` - nor with a `main` HTML element. You're not constrained to use this together with `AppMain`, but rather, you can use it in whichever layout you prefer - it's completely flexible and up to your particular case. And this is also the reason why I wanted to keep the example as simple and implementation-agnostic as possible: I didn't want to generate confusion about what this specific component is about, nor dictate how to use it, but I get your point. I will add some basic layout in the example that clarifies what they user needs to do in order to see the skip link in action :)

Reply:
Yeah I agree with you, I guess my two comments is a little conflicting haha. I do think then having `href` as a prop for `SkipLink` makes sense to make it a bit more flexible. In terms of the documentation, I think maybe we can say it "could" be used with `AppMain` instead of it "must" if that makes sense?